layouts/shortcodes/runnable.html

```diff
@@ -14,8 +14,8 @@
                   <nav class="nav-languages">
                     <a class="language active" data-action="nav-lang" data-target="query" href="#">Query</a>
                     <a class="language" data-action="nav-lang" data-target="curl" href="#">Curl</a>
-                    <a class="language" data-action="nav-lang" data-target="go" href="#">Go</a>
-                    <a class="language" data-action="nav-lang" data-target="java" href="#">Java</a>
+                    <a class="language" data-action="nav-lang" data-target="go-grpc" href="#">Go (grpc)</a>
```

Comment:
Go grpc example doesn't work on master because we changed the path to protocol buffer files. So it is
`"github.com/dgraph-io/dgraph/protos"` instead of 
`"github.com/dgraph-io/dgraph/protos/graphp"` but we shouldn't change it for now because latest release still uses the `graphp`.

Makes we wonder if this mock code should be passed from the docs to the theme. 

Another issue is that is it possible to have custom grpc code for some examples?
We need that for https://docs.dgraph.io/v0.7.6/query-language/#graphql-variables because the method we wan't to use is `SetQueryWithVariables` instead of just `SetQuery`.